core/types/block.go

```diff
@@ -516,6 +516,7 @@ func (h *Header) SetExcessDataGas(v *big.Int) {
 		h.ExcessDataGas.Set(v)
 	}
 	if h.WithdrawalsHash == nil {
+		fmt.Println("Setting withdrawals hash to empty root hash")
```

Comment:
curious, no logging library is used and we use `fmt.Println` for logging?

Reply:
I see log / fmt.Println being used both 🤔

Reply:
Good catch I meant to remove this (it's from my debugging.) I will remove it.